lib/controllers/ServiceBrokerApiController.js

```diff
@@ -15,6 +15,7 @@ const ServiceInstanceNotFound = errors.ServiceInstanceNotFound;
 const ServiceBindingAlreadyExists = errors.ServiceBindingAlreadyExists;
 const ServiceBindingNotFound = errors.ServiceBindingNotFound;
 const ContinueWithNext = errors.ContinueWithNext;
+const CONST = require('../constants');
 
 class ServiceBrokerApiController extends BaseController {
```

Comment:
Can you also change the minimum broker API version? CC now sends 2.13 so it is good to update it.

Reply:
Done